editor/editor_spin_slider.cpp

```diff
@@ -111,7 +114,21 @@ void EditorSpinSlider::_gui_input(const Ref<InputEvent> &p_event) {
 			}
 
 			if (grabbing_spinner) {
+				// Don't make the user scroll all the way back to 'in range' if they went off the end
```

Comment:
We're about to settle on a comment style close to this, but ending sentences with a dot. Could you add the final dot here and on line 126?